packages/ActionBar/stories/ShowcaseApp/MyColumns.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { ColumnsArrangement, useColumnsArragment } from "../../src";
+
+const defaultColumnsForArrangement = [
```

Comment:
I would suggest that the defaultColumnArragement be an array of IDs if we pass the full objects when you are developing will be hard to move objects around to create a specific state you might want to test.

Ex.

its easier to modify `['goals', 'names', 'status', 'country']` and move it around than an entire object, now Imagine that you have a 100 rows in a json format, would be annoying to have to move that data structure around just specify what order you want for your columns.

I struggle with this when I was doing the POF for Assesst and Attributes

Reply:
How about this?
```js
<ColumnsArrangement
      orderedColumnIds={["goals", "names", "status", "country"]}
      onChangeOrder={() => {}}
      onHideAll={() => {}}
      onShowAll={() => {}}
      onChangeVisibility={() => {}}
    >
      <ColumnsArrangement.ColumnDefinition id="goals" label="Goals" isDisabled />
      <ColumnsArrangement.ColumnDefinition id="name" label="Name" isHidden />
      <ColumnsArrangement.ColumnDefinition id="status" label="Status" />
      <ColumnsArrangement.ColumnDefinition id="country" label="Country" />
</ColumnsArrangement>
```